### DISCUSSION TITLE
Access of private wallet

### DISCUSSION THREAD

**Discussion by hmercker** [catchthatrabbit/coreminer#59] [Wishlist]
After having generated a wallet, can someone please outline the procedure how to access ones wallet with the private key?

**Comment by rastislavcore** [comment]
The best access is using the official go-core client and console.
You can download it on the following link: https://github.com/core-coin/go-core/releases
I did an example of how to transfer funds here: https://gist.github.com/rastislavcore/9a608cc79fa78c5209fbf4bbbd5eaec0

**Reply by hmercker** [reply]
Thank you for your quick response. 
Is there a beginners guide somewhere posted already? I've tried the procedure but it gets stuck along the way. Something I must be doing wrong.

**Reply by rastislavcore** [reply]
Make sure you are using the right network.
More info about wallet generator here: https://github.com/core-coin/wallet-generator#core-blockchain-ican-wallet-generator

**Reply by hmercker** [reply]
Sorry Raisty, I feel completely computer illiterate. I just cannot get it going. Can you please set up a guide, similar to "start the core miner"?

**Reply by mrmike15** [reply]
rastislavcore tried your fund transfer link and it says page not found...

**Reply by rastislavcore** [reply]
@mrmike15 I apologize, it was outdated. Now fixed.

**Comment by zdh9966** [comment]
what's this

在 2023年7月23日星期日，mrmike15 ***@***.***> 写道：
